tools/utils/BPE.lua

```diff
@@ -38,9 +52,15 @@ local function getPairs(word)
   return pairs
 end
 
-local function str2word(l)
+local function str2word(l, case_insensitive)
```

Comment:
It's a function only used for BPE and does not have any unicode functionality. Is it really necessary to move it into unicode.lua?